ansible/README.md

```diff
@@ -149,6 +149,52 @@ You need to run `initdb` on cloudant **only once** per cloudant database to init
 **Hint:** The `initdb.yml` playbook will only initialize your database if it is not initialized already, else it will skip initialization steps.
 
 
+### Verification after Deployment
```

Comment:
This is repeating some steps from #784 - we should consolidate. @csantanapr thoughts?